src/services/to-dos.service.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <json/json.h>
+
+struct ToDo {
```

Comment:
That structure looks like [DTO](https://learn.microsoft.com/en-us/aspnet/web-api/overview/data/using-web-api-with-entity-framework/part-5). If its true please name that `ToDoDto` and store separatly in folder `dtos`